cypher-shell/src/integration-test/java/org/neo4j/shell/MainIntegrationTest.java

```diff
@@ -189,6 +178,60 @@ public void promptsOnPasswordChangeRequired() throws Exception {
         }
     }
 
+    @Test
+    public void allowUserToUpdateExpiredPasswordInteractivelyWithoutBeingPrompted() throws Exception {
+        //given a user that require a password change
+        int majorVersion = getVersionAndCreateUserWithPasswordChangeRequired();
+
+        //in 4.0 and later when the user attempts a non-interactive password update
+        if (majorVersion >= 4 ) {
```

Comment:
You could use `assume` here, since this test has no value if running with neo4j 3.5 or older.